src/engines/name.go

```diff
@@ -23,6 +26,40 @@ const (
 	YEP
 )
 
+func initPrettyName() {
+	// laid out like this instead of ["Undefined", "Bing", "Brave", ...] so the index doesn't matter
+	PrettyName = make([]string, len(_NameValues))
+	PrettyName[UNDEFINED] = "UNDEFINED"
+	PrettyName[BING] = "Bing"
+	PrettyName[BRAVE] = "Brave"
+	PrettyName[DUCKDUCKGO] = "DuckDuckGo"
+	PrettyName[ETOOLS] = "Etools"
+	PrettyName[GOOGLE] = "Google"
+	PrettyName[GOOGLESCHOLAR] = "GoogleScholar"
+	PrettyName[MOJEEK] = "Mojeek"
+	PrettyName[PRESEARCH] = "Presearch"
+	PrettyName[QWANT] = "Qwant"
+	PrettyName[STARTPAGE] = "Startpage"
+	PrettyName[SWISSCOWS] = "Swisscows"
+	PrettyName[YAHOO] = "Yahoo"
+	PrettyName[YEP] = "Yep"
+
+	// Check if all search engines have a pretty name set
+	for _, eng := range NameValues() {
+		if PrettyName[eng] == "" {
+			log.Fatalf("engines.init() (names.go): %v doesn't have a pretty name set.", eng)
+			// ^FATAL
+		}
+	}
+}
+
+func init() {
```

Comment:
This is a runtime errors, we must check for it before building our images otherwise it could result in broken version being pushed. I managed to do this in my PR that tries to add pretty names.